nextjs/pages/index.tsx

```diff
@@ -2,18 +2,24 @@ import React, { useState } from "react";
 import WebcamCapture from "./components/WebcamCapture";
 import SignTutorial from "./components/SignTutorial";
 import ReactPlayer from "react-player";
+import ToggleButton from "./components/ToggleButton";
 
 export default function LearningPage() {
 
+    const buttonCSS = "bg-blue-500 hover:bg-green-700 text-white font-bold py-2 px-4 rounded";
+
     const [showWebcam, setShowWebcam] = useState(false)
     const [showSignTutorial, setShowSignTutorial] = useState(false);
 
     const closeSignTutorial = () => setShowSignTutorial(false);
     const displaySignTutorial = () => setShowSignTutorial(true);
 
-    const buttonCSS = "bg-blue-500 hover:bg-green-700 text-white font-bold py-2 px-4 rounded";
     const [[currentSign, URL], setCurrentSign] = useState(getNextSign());
 
+    /* Should analyse toggle switch state*/
+    const [shouldAnalyse, setShouldAnalyse] = useState(false);
+    const onToggle = () => setShouldAnalyse(!shouldAnalyse);
```

Comment:
Hvad bliver den her brugt til?

Reply:
Hvis du har den toggled så sender den videoen til at blive gemt på serveren i stedet for til mediapipe lige pt